react-app/src/library/types/consumed-headset-events.ts

```diff
@@ -11,7 +11,6 @@ export enum HeadsetEvents {
     loggableEvent = 'loggableEvent',
     webHidPermissionRequested = 'webHidPermissionRequested',
     deviceConnectionStatusChanged = 'deviceConnectionStatusChanged',
-    deviceMuteChanged = 'deviceMuteChanged',
```

Comment:
Is it ok to just remove this? It does need to be renamed like the other event?

Reply:
there's a `deviceMuteStatusChanged` event already. We were using `deviceMuteChanged` for mutes and `deviceHoldStatusChanged` for holds. The `deviceMuteStatusChanged` seemed to be unused so I switched over to that one to match the naming style for holds and deleted the old one so we only have a single mute event in the types.

Reply:
👍🏼